src/components/PartnersContent/PartnersTile/PartnersTile.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FunctionComponent } from "react";
+import { Button, ButtonSize } from "@govtechsg/tradetrust-ui-components";
+import { PartnersTileProps } from "../types";
+
+export const PartnersTile: FunctionComponent<PartnersTileProps> = ({ data }) => {
+  const { companyName, description, logo, websiteLink } = data;
+  return (
+    <div className="max-w-[380px] min-w-[380px] bg-white rounded-xl shadow-md p-5 mb-10 mr-8">
+      <p className="text-2xl">{companyName}</p>
+      <div className="max-h-[148px] min-h-[148px] flex justify-center">
+        <img src={logo} className="mx-auto" data-testid={`partner-logo-${companyName}`} />
+      </div>
+      <div className="mb-12 max-h-[152px] min-h-[152px] break-words">{description}</div>
+      <div className="flex justify-center">
```

Comment:
👀

Reply:
?

Reply:
i also wanna fix this height so that it will all look the same

Reply:
ok this is fine. updated